### PR TITLE
B184- fixed numSongs never goes back to 0 and added the updated delete function to create a queue

### PR DESCRIPTION
**Pull request by JohnnyPersichilli**

### CODE REVIEW

File: CranberryQueue/Queue/SongTableView.swift [hunk 1]
```diff
@@ -54,6 +54,7 @@ class SongTableView: UITableView, UITableViewDelegate, UITableViewDataSource, Qu
             }
             if snap.documents.count == 0 {
                 DispatchQueue.main.async {
+                    self.songDelegate?.updateNumSongs(0)
```
Comment (rolflocher) [inline CranberryQueue/Queue/SongTableView.swift]: called it. nice

File: CranberryQueue/Map/MapViewController.swift [hunk 2]
```diff
@@ -172,18 +172,6 @@ class MapViewController: UIViewController, mapDelegate, UITextFieldDelegate, Log
             if(!isHost){
                 self.db?.collection("contributor").document(self.queueId!).collection("members").document(self.uid).delete()
             }else{
-                self.db?.collection("contributor").document(self.queueId!).delete()
-                
-                self.db?.collection("song").whereField("queueId", isEqualTo: self.queueId).getDocuments(completion: { (snapshot, err) in
-                    guard let snap = snapshot else {
-                        return
-                    }
```
Comment (rolflocher) [inline CranberryQueue/Map/MapViewController.swift]: Should joinQueue be updated also?

Comment (JohnnyPersichilli) [reply CranberryQueue/Map/MapViewController.swift]: tony missed this when he updated his function to replace all the delete methods with just the delete from locations table that triggers his other function

Comment (rolflocher) [reply CranberryQueue/Map/MapViewController.swift]: alright

Comment (creiser2) [reply CranberryQueue/Map/MapViewController.swift]: I think this diff was already merged

Comment (creiser2) [reply CranberryQueue/Map/MapViewController.swift]: this diff was merged with master

**Pull request merged**